presto-benchmark-runner/src/test/java/com/facebook/presto/benchmark/retry/TestRetryDriver.java

```diff
@@ -66,21 +65,10 @@ public Integer run()
 
     private RetryDriver retryDriver;
 
-    @BeforeMethod
-    public void setup()
```

Comment:
I don't think this resolves the issue which is that multiple tests use the same RetryDriver in a multi-threaded test?

I would fix this by removing the member variable and constructing the retry driver in each test on the stack so it is thread safe and fast.

RetryDriver does look like it is probably thread safe, but It isn't  documented as such so I would treat it as if it isn't in this test.

Reply:
Yes, retryDriver Should be defined locally. I'll change it. Thanks